exporters/otlp/internal/partialsuccess_test.go

```diff
@@ -36,9 +36,8 @@ func requireErrorString(t *testing.T, expect string, err error) {
 }
 
 func TestPartialSuccessFormat(t *testing.T) {
-	requireErrorString(t, "empty message (0 metric data points rejected)", PartialSuccessToError(MetricsPartialSuccess, 0, ""))
-	requireErrorString(t, "help help (0 metric data points rejected)", PartialSuccessToError(MetricsPartialSuccess, 0, "help help"))
-	requireErrorString(t, "what happened (10 metric data points rejected)", PartialSuccessToError(MetricsPartialSuccess, 10, "what happened"))
-	requireErrorString(t, "what happened (15 spans rejected)", PartialSuccessToError(TracingPartialSuccess, 15, "what happened"))
-	requireErrorString(t, "empty message (7 log records rejected)", PartialSuccessToError("log records", 7, ""))
+	requireErrorString(t, "empty message (0 metric data points rejected)", MetricPartialSuccessError(0, ""))
```

Comment:
Doesn't this same change need to be made for the [metric](https://github.com/open-telemetry/opentelemetry-proto/blob/724e427879e3d2bae2edc0218fff06e37b9eb46e/opentelemetry/proto/collector/metrics/v1/metrics_service.proto#L58-L60) exporter?

Reply:
It does, but we haven't implemented partial response handling there yet. I'm happy to add it in a follow on PR though.

FWIW, it is being tracked here: https://github.com/open-telemetry/opentelemetry-go/issues/3162